api/controllers/user.js

```diff
@@ -242,8 +242,22 @@ async function getUser(req, res) {
   }
 }
 
+const UPDATEABLE_FIELDS = [
+  'email',
```

Comment:
@martinbedouret Now that I think about it, it might be problematic to let users change their email address, because we currently use the `email` property as a foreign key for communicators and boards. We might want update both schemas to use a user id instead.

Reply:
I created an issue for this: https://github.com/cboard-org/cboard-api/issues/140.

Reply:
you are right. To prevent issues, we should block the front end to update the email until this is implemented.

Reply:
issue for the frontend: https://github.com/cboard-org/cboard/issues/866